proposals/description/fip_97.ts

```diff
@@ -0,0 +1,34 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_97: ProposalDescription = {
+  title: 'FIP-97: Seed Turbo fuse pool with 10M Fei',
+  commands: [
+    {
+      target: 'fei',
+      values: '0',
+      method: 'transfer',
+      arguments: ['{turboFusePCVDeposit}', '10000000000000000000000000'], // 10M
+      description: 'Transfer $10M FEI from optimistic timelock to the Turbo Fuse PCV deposit'
+    },
+    {
+      target: 'turboFusePCVDeposit',
+      values: '0',
+      method: 'deposit()',
+      arguments: [],
+      description: 'Deposit ~$10M of Fei into Turbo Fuse Pool'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'addDeposits(address[])',
+      arguments: [['{turboFusePCVDeposit}']],
```

Comment:
Ah. You'll need to do a wrapper because the PCV deposit doesn't accurately report FEI